scenarioo-client/app/shared/navigation/branchOrderBy.filter.js

```diff
@@ -40,6 +43,40 @@ function scBranchOrderByFilter() {
             return 1;
         }
 
```

Comment:
aha, moment, I got it, you are doing it right I think - so for the build date comparison you are still ordering the aliases ascending by name, and only for alphabetical ordering you are also ordering the aliases with the same rule (ascending or descending) - right? In that case that looks ok to me.